tasks/test-project/util.js

```diff
@@ -30,6 +30,7 @@ const getExecaOptions = (cwd) => ({
   cwd,
   env: {
     RW_PATH: path.join(__dirname, '../../'),
+    REDWOOD_CI: '1',
```

Comment:
tasks/test-project is used to build a local, functional project via `yarn build:test-project`. It's **not** used with either local E2E or CI E2E.

I would suggest including test-project building in telemetry data instead of filtering out.

Reply:
It's still being sent in, there's just a field for `redwoodCi` that's set to `true` when we're the ones sending in telemetry data:

![image](https://user-images.githubusercontent.com/300/148142387-3e912e9c-b0f6-4db8-bcfc-f79482f0b180.png)